src/components/OSCALBackMatter.js

```diff
@@ -47,7 +48,9 @@ function DescriptionDisplay(props) {
     );
   }
   return (
-    <StyledTooltip title={props.resource.description}>
+    <StyledTooltip
+      title={<OSCALMarkupLine text={props.resource.description} />}
```

Comment:
`description` is `markup-multiline`: https://pages.nist.gov/OSCAL/reference/latest/catalog/json-reference/#/catalog/back-matter/resources/description